build_artifacts/v1/v1.0/v1.0.0/v1.0.0-beta/cpu.env.in

```diff
@@ -20,3 +20,7 @@ conda-forge::boto3[version='>=1.28.35,']
 conda-forge::sagemaker-python-sdk[version='>=2.173.0,']
 conda-forge::supervisor[version='>=4,']
 conda-forge::autogluon[version='>=0.8.2,']
+conda-forge::sagemaker-studio-sparkmagic-lib[version='>0.1.2,<1']
```

Comment:
Both `sagemaker-studio-sparkmagic-lib` and `sparkmagic` are direct downstream dependencies of `sagemaker-studio-analytics-extension` correct?

I suggest removing them from the env file, instead allowing the conda resolver to bring in the correct versions as dictated by the constraints of `sagemaker-studio-analytics-extension`

Reply:
Sure let me check and remove dependencies accordingly